documentation/tutorials/Demo_1_Basic_Run/run_pecan.qmd

```diff
@@ -65,6 +65,9 @@ install.packages('PEcAn.all')
 
 - A valid `pecan.xml` configuration file or use the example provided: `pecan/documentation/tutorials/Demo_1_Basic_Run/pecan.xml`
 
+- Some steps in this notebook are model-specific and require the corresponding PEcAn model package to be installed (e.g. `PEcAn.SIPNET` when using the SIPNET model, which can be installed by running: `install.packages('PEcAn.SIPNET')`).
```

Comment:
This note won't be needed at all if you edit line 62 above to be `install.packages(c("PEcAn.all", "PEcAn.SIPNET"))`